lib/lbt/analyzer/XMLTemplateAnalyzer.js

```diff
@@ -350,6 +353,54 @@ class XMLTemplateAnalyzer {
 					// console.log("child view detected %s", childViewModuleName);
 					this._addDependency(childViewModuleName, conditional);
 				}
+			} else if ( moduleName === TEMPLATEVIEW_MODULE ) {
+				const viewName = getAttribute(node, ANYVIEW_VIEWNAME_ATTRIBUTE);
+				if ( viewName ) {
+					const childViewModuleName = fromUI5LegacyName( viewName, ".view.tmpl" );
+					// console.log("child view detected %s", childViewModuleName);
+					this._addDependency(childViewModuleName, conditional);
+				}
+			} else if ( moduleName === VIEW_MODULE ) {
+				const viewName = getAttribute(node, ANYVIEW_VIEWNAME_ATTRIBUTE);
+				if ( viewName ) {
+					let childViewModuleName;
+
+					if (viewName.startsWith("module:")) {
+						childViewModuleName = viewName.slice("module:".length) + ".js";
+					} else {
+						const viewType = getAttribute(node, VIEW_TYPE_ATTRIBUTE);
+
+						let viewTypeExtension;
+
+						switch (viewType) {
+						case "JS":
+							viewTypeExtension = ".view.js";
+							break;
+						case "JSON":
+							viewTypeExtension = ".view.json";
+							break;
+						case "Template":
+							viewTypeExtension = ".view.tmpl";
+							break;
+						case "XML":
+							viewTypeExtension = ".view.xml";
+							break;
+						case "HTML":
+							viewTypeExtension = ".view.html";
+							break;
+						default:
+							log.warn(`Unable to analyze sap.ui5/rootView: Unknown type '${viewType}'`);
+						}
+
+						if (viewTypeExtension) {
+							childViewModuleName = fromUI5LegacyName(viewName, viewTypeExtension);
+						}
+					}
+					if (childViewModuleName) {
+						// console.log("child view detected %s", childViewModuleName);
```

Comment:
log verbose, silly or remove?

Reply:
I kept it consistent with the existing code. Should be addressed in general. I guess we have several places where console.log is commented out.